### PR TITLE
Fix #3563 : Merge profile_progress_fragment.xml into single xml file

### LINKED ISSUE #3563: Merge profile_progress_fragment.xml into single xml file
Currently there are 4 versions of `profile_progress_fragment.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by bhaktideshmukh**
## Explanation
Fix #3563 : Merge profile_progress_fragment.xml into single xml file

Phone portrait mode

| Before  | After|
|-------|---|
|||

Phone landscape mode

| Before  | After|
|-------|---|
|||

Tablet portrait mode

| Before  | After|
|-------|---|
|||
 
Tablet landscape mode

| Before  | After|
|-------|---|
|||

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -467,6 +467,11 @@
   <dimen name="topic_practice_footer_view_margin_top">36dp</dimen>
   <dimen name="topic_practice_footer_view_margin_bottom">108dp</dimen>
 
+  <!-- Profile Progress Fragment -->
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: Better write them as `ProfileProgressFragment` everywhere.

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -467,6 +467,11 @@
...
+  <dimen name="profile_progress_fragment_marginStart">20dp</dimen>
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: Rename to `profile_progress_fragment_margin_start `
here and everywhere else. All names should be `small_case` separated by `_`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@bhaktideshmukh Just took a quick look. The screenshots of before/after do not match in terms of background color. Please check that.

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (bhaktideshmukh) [reply app/src/main/res/values/dimens.xml]: Done!

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (bhaktideshmukh) [reply app/src/main/res/values/dimens.xml]: Done.

**Comment by bhaktideshmukh** [comment]
@rt4914 Hey I've used android:background="@color/transparent" instead of android:background="@color/white" Since in background color white , fragments of after commit weren't matching the fragments of before commit.

**Comment by bhaktideshmukh** [comment]
PTAL @rt4914 
Thanks.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**